qiskit/circuit/__init__.py

```diff
@@ -282,6 +282,18 @@
    Operation
    EquivalenceLibrary
 
+Annotated Operations
+''''''''''''''''''''
```

Comment:
Why not `----------------------`? You use that on stable/0.46

https://github.com/Qiskit/qiskit/blob/1e0e4fccd469617489a28e4cfa38271a9b0b617b/qiskit/circuit/__init__.py#L363-L364

Reply:
Oh, I didn't realize that we had fixed this on `stable/0.45`. I used this here to add a third level of hierarchy a subsubsection under `Gates and Instructions` which seemed like a better fit for the structure here. But if you think it's better with the flatter hierarchy I can switch this over to match that.

Reply:
Flatter hierarchy will work better with iqp-channel-docs. Generally it works best with H1 - H3, and h4-h6 aren't very meaningful.

Reply:
Ok, I'll update it to use `----------------------` then:

```suggestion
--------------------
```